backend/wmg/pipeline/cell_type_ancestors.py

```diff
@@ -30,8 +30,7 @@ def create_cell_type_ancestors(corpus_path: str) -> None:
     valid_cell_types = cell_counts_df["cell_type_ontology_term_id"].unique()
     ancestors_dict = {}
     for cell_type_id in valid_cell_types:
-        cell_type_ancestors = sorted(set(ancestors(cell_type_id)).intersection(valid_cell_types))
-        ancestors_dict[cell_type_id] = ",".join(cell_type_ancestors)
+        ancestors_dict[cell_type_id] = sorted(set(ancestors(cell_type_id)).intersection(valid_cell_types))
```

Comment:
the pipeline changes that were previously merged in incorrectly stored the ancestors as a comma-separated string, but we need it as a python list.